### PR TITLE
Update README.md with new documentation

### PR DESCRIPTION
**Pull request by muellerfr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,31 @@
...
+#### Visit our [Web Site](https://aws-amplify.github.io) to learn more about AWS Amplify.
 
-Please note the AWS SDK for Android supports Android API level 10+ and newer libraries may require higher API level 21+ and 23+.
-
-### Sign up for AWS
-Before you begin, you need to sign up for an AWS account [here](http://aws.amazon.com/), click 'Sign In to the Console', and select new user.
-
-### Create an Amazon Cognito Identity pool 
-Amazon Cognito Identity allows you to authenticate users to access your AWS resources without having to place your credentials within the applicaiton itself (which is *very* insecure).
+* [Documentation](https://aws-amplify.github.io/docs)
```
Comment (minbi) [inline README.md]:
```suggestion
* [Documentation](https://aws-amplify.github.io/docs/android/start)
```

File: README.md [hunk 2]
```diff
@@ -251,36 +180,14 @@ If you are using a Windows machine, unit tests that involve file deletion can fa
...
-* [Amazon Security Token Service (STS)](https://docs.aws.amazon.com/STS/latest/APIReference/Welcome.html)
-* [Amazon Simple Email Service (SES)](https://aws.amazon.com/ses/)
-* [Amazon Simple Notification Service (SNS)](https://aws.amazon.com/sns/)
-* [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/)
-* [Amazon SimpleDB](https://aws.amazon.com/simpledb/)
-* [Amazon Transcribe](https://aws.amazon.com/transcribe/)
-* [Amazon Comprehend](https://aws.amazon.com/comprehend)
+## Talk to Us
+
+Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-ios/issues) to leave feedback and to connect with other users of the SDK.
```
Comment (minbi) [inline README.md]:
```suggestion
Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
```

File: README.md (hunk 1 above)
Comment (muellerfr) [reply README.md]: By Design

File: README.md (hunk 2 above)
Comment (muellerfr) [reply README.md]: Fixed

**Pull request merged**